src/basic/special.h

```diff
@@ -116,3 +114,9 @@
 #define SPECIAL_SESSION_SLICE "session.slice"
 #define SPECIAL_APP_SLICE "app.slice"
 #define SPECIAL_BACKGROUND_SLICE "background.slice"
+
+/* PREFIX for expanding the quota service unit templates */
+#define SPECIAL_QUOTACHECK_PREFIX "systemd-quotacheck"
+#define SPECIAL_QUOTACHECK_NETWORK_PREFIX "systemd-quotacheck-network"
```

Comment:
please don't define this to a "prefix", instead just define this to the template name, and then instantiate it via unit_name_replace_instance()

Reply:
what's the point of the separate "network" services? that makes no sense to me? if the mount itself is already established the quota service should not need additional deps.

Reply:
Actually, I wanted to distinguish between local filesystems, that would get ordered below local-fs.target and network provided file systems, that would get ordered below remote-fs.target.
But it seems that ordering can also be covered by a generator_add_ordering function like in systemd-growfs.
So, I will use that.